pkg/config/config.go

```diff
@@ -7,27 +7,36 @@ import (
 var (
 	accessTokenField = field.StringField(
 		"token",
+		field.WithDisplayName("Personal access token"),
 		field.WithDescription("The GitHub access token used to connect to the GitHub API."),
+		field.WithRequired(true),
+		field.WithIsSecret(true),
 	)
 	orgsField = field.StringSliceField(
 		"orgs",
+		field.WithDisplayName("Organizations"),
 		field.WithDescription("Limit syncing to specific organizations."),
 	)
 	instanceUrlField = field.StringField(
 		"instance-url",
+		field.WithDisplayName("GitHub instance URL"),
 		field.WithDescription(`The GitHub instance URL to connect to. (default "https://github.com")`),
 	)
 	appIDField = field.StringField(
 		"app-id",
+		field.WithDisplayName("GitHub App ID"),
 		field.WithDescription("The GitHub App to connect to."),
 	)
 
 	appPrivateKeyPath = field.StringField(
 		"app-privatekey-path",
+		field.WithDisplayName("GitHub App private key (.pem)"),
 		field.WithDescription("Path to private key that is used to connect to the GitHub App"),
+		field.WithIsSecret(true),
```

Comment:
Let's remove `WithIsSecret` from here for now, not sure how we're going to handle files yet